fast/stages/1-resman/moved/v35.1.0-v36.0.0.tf

```diff
@@ -236,3 +236,21 @@ moved {
   from = module.branch-gcve-prod-r-sa[0]
   to   = module.stage3-sa-ro["gcve-prod"]
 }
+
+import {
+  for_each = var.fast_stage_2.network_security.enabled ? [""] : []
+  to       = module.nsec-organization-roles[0].google_organization_iam_custom_role.roles["network_firewall_policies_admin"]
+  id       = "organizations/${var.organization.id}/roles/networkFirewallPoliciesAdmin"
+}
+
+import {
+  for_each = var.fast_stage_2.network_security.enabled ? [""] : []
+  to       = module.nsec-organization-roles[0].google_organization_iam_custom_role.roles["ngfw_enterprise_admin"]
+  id       = "organizations/${var.organization.id}/roles/ngfwEnterpriseAdmin"
+}
+
+import {
+  for_each = var.fast_stage_2.network_security.enabled ? [""] : []
+  to       = module.nsec-organization-roles[0].google_organization_iam_custom_role.roles["ngfw_enterprise_viewer"]
+  id       = "organizations/${var.organization.id}/roles/ngfwEnterpriseViewer"
+}
```

Comment:
The migration scenario then assumes following actions:
* run `terraform state rm ...` on 0-bootstrap - to remove roles from state and prevent their deletion on `apply`
* run `terraform apply` on 1-resman

I guess, to keep the migration paths clean, we need to call out that manual step somewhere.

Reply:
Correct. I tried adding a `removed` block  to bootstrap but couldn't figure it out.